### PR TITLE
Update READMEs

### PR DESCRIPTION
**Pull request by RomanTsukanov**

### CODE REVIEW

**Review by andrewtelnov — APPROVED** [review]

**Review by tsv2013 — APPROVED** [review]

**Review by tsv2013 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,79 +1,35 @@
...
-<a href="https://www.npmjs.com/package/survey-creator"><img alt="NPM Version" src="https://img.shields.io/npm/v/survey-creator.svg" data-canonical-src="https://img.shields.io/npm/v/survey-creator.svg" style="max-width:100%;"></a>
 <a href="https://github.com/surveyjs/survey-creator/issues">
 <img alt="Issues" title="Open Issues" src="https://img.shields.io/github/issues/surveyjs/survey-creator.svg">
 </a>
 <a href="https://github.com/surveyjs/survey-creator/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aclosed+">
 <img alt="Closed issues" title="Closed Issues" src="https://img.shields.io/github/issues-closed/surveyjs/survey-creator.svg">
 </a>
 
-Survey Creator is a visual survey designer / form builder for [SurveyJS](https://github.com/surveyjs/survey-library). It uses JSON to store survey metadata.
+Survey Creator is a visual survey designer / form builder for [SurveyJS](https://github.com/surveyjs/survey-library).
```
Comment (arminal) [inline README.md:10]:
```suggestion
The Survey Creator is a visual survey designer / form builder for [SurveyJS](https://github.com/surveyjs/survey-library).
```

File: README.md [hunk 2]
```diff
@@ -1,79 +1,35 @@
...
+- [Documentation](https://surveyjs.io/Documentation/Survey-Creator)
+- [Create your Survey or Form now](https://surveyjs.io/create-survey/)
 
-    ```
-    npm run build_prod
-    ```
+## Survey Creator V2 (Beta)
 
-    You can find the built library in the "build" directory.
+Survey Creator V2 is a successor of V1. It has an updated design and separates model from the UI.
```
Comment (arminal) [inline README.md:24]: separates what model?

File: packages/survey-creator-react/README.md [hunk 3]
```diff
@@ -1,78 +1,67 @@
...
-
-### Read documentation
-
-To find out more about the library concepts and the you may customize it, please read [our documentation](https://surveyjs.io/Documentation/Survey-Creator)
+```
 
 ## License
 
-The Survey Creator (Form Builder) is NOT FREE for commercial using. Please find more about licensing the product [here](http://surveyjs.io/Licenses).
+The Survey Creator (Form Builder) is NOT FREE for commercial usage. Please find more about licensing the product on the following page: [Licenses](http://surveyjs.io/Licenses).
```
Comment (arminal) [inline packages/survey-creator-react/README.md:67]:
```suggestion
The Survey Creator (Form Builder) is **not available for free commercial usage**. You can find more information about licensing this product on the following page: [Licenses](http://surveyjs.io/Licenses).
```

File: packages/survey-creator/README.md [hunk 4]
```diff
@@ -1,84 +1,77 @@
-**survey.creator** is the visual survey creator / form builder for [SurveyJS](https://github.com/surveyjs/survey-library). It uses JSON to store the survey metadata.
+# Survey Creator V1
 
 [![Build Status](https://dev.azure.com/SurveyJS/SurveyJS%20Integration%20Tests/_apis/build/status/SurveyJS%20Creator?branchName=master)](https://dev.azure.com/SurveyJS/SurveyJS%20Integration%20Tests/_build/latest?definitionId=8&branchName=master)
 <a href="https://www.npmjs.com/package/survey-creator"><img alt="NPM Version" src="https://img.shields.io/npm/v/survey-creator.svg" data-canonical-src="https://img.shields.io/npm/v/survey-creator.svg" style="max-width:100%;"></a>
 
-## See Survey Creator / Form Builder in action
+Survey Creator is a visual survey designer / form builder for [SurveyJS](https://github.com/surveyjs/survey-library).
```
Comment (arminal) [inline packages/survey-creator/README.md:6]:
```suggestion
The Survey Creator is a visual survey designer / form builder for [SurveyJS](https://github.com/surveyjs/survey-library).
```

File: packages/survey-creator/README.md [hunk 5]
```diff
@@ -1,84 +1,77 @@
...
-## Dependencies
-
-The library depends on: [surveyjs](http://surveyjs.io/Library/) and [knockoutjs](http://knockoutjs.com)
-Optionally you may use [bootstrap](http://getbootstrap.com), [ace editor](https://ace.c9.io/) and [select2](https://select2.org/)
+    This command runs unit tests using [Karma](https://karma-runner.github.io/0.13/index.html).
 
 ## License
 
-The Survey Creator (Form Builder) is NOT FREE for commercial using. Please find more about licensing the product [here](http://surveyjs.io/Licenses).
+The Survey Creator (Form Builder) is NOT FREE for commercial usage. Please find more about licensing the product on the following page: [Licenses](http://surveyjs.io/Licenses).
```
Comment (arminal) [inline packages/survey-creator/README.md:77]:
```suggestion
The Survey Creator (Form Builder) is **not available for free commercial usage**. You can find more information about licensing this product on the following page: [Licenses](http://surveyjs.io/Licenses).
```

**Comment by arminal** [comment]
#GrammarReview

Article and phrasing edits.

**Review by andrewtelnov — APPROVED** [review]

**Pull request merged**